lambda/voting/scripts/utils.mjs

```diff
@@ -0,0 +1,54 @@
+//Helper functions for the MetaKeep APIs
+
+import fetch from "node-fetch";
+
+export default async function getDeveloperWallet() {
```

Comment:
copy this again. I had added result.ok check.

Same for the transaction status function below

Reply:
done